spec/controllers/orgs/google_classroom_configurations_controller_spec.rb

```diff
@@ -53,8 +53,7 @@
           end
 
           it "flashes error message" do
-            message = "A LMS configuration already exists. Please remove configuration before creating a new one."
-            expect(flash[:alert]).to eq(message)
+            expect(flash[:alert]).to be_present
```

Comment:
In these tests, why aren't we able to assert exact messages anymore? This loses the assertion that we're displaying the correct error.

Reply:
Oh I can add those back! I didn’t think about it that way, but that makes sense.